docs/components/ComponentsWrapper.tsx

```diff
@@ -0,0 +1,26 @@
+import { useTheme } from "nextra-theme-docs";
+import Logo from "./Logo";
+import Splash from "./Splash";
+import WarningBox from "./WarningBox";
+
+export default function ComponentsWrapper() {
```

Comment:
is there a better name for this? `IndexIntro` or something

Reply:
Done, updated to IndexIntro